tests/test_flumpy.py

```diff
@@ -5,7 +5,7 @@
 import numpy as np
 import pytest
 
-import scitbx.array_family.flex as flex
+import cctbx.array_family.flex as flex
```

Comment:
same here

Reply:
We rely in _many_ places on this behaviour, so I don't see why it is an issue here? It's highly unlikely to deviate from being a superset, and if it does will likely break things in many more places than just here.

Reply:
The entire point of this is to avoid cctbx/scitbx implicit behaviour